### DISCUSSION TITLE
Multimode

### DISCUSSION THREAD

**Discussion by dejurin** [philippta/flyscrape#72] [Ideas]
I really enjoyed the implementation of the programme. Thank you for your project.
I have a question, is it possible to add scraping modes to the programme? 
This is what I mean: now we can run one configuration file, is it possible to add **multimode**? What is its usefulness? Sometimes it happens that we need to scrape many sites to get data for comparison. For example, the price of an iPhone at 20 different shops. In fact, we need to go through 20 pages with different scraping configurations. Running 10, 20, 100 programs per one configuration is a tedious task.
And that's where **multimode** comes to the rescue with its powerful implementation of concurrency, proxies, and etc.
What do you think about it, and how is it possible to implement this mode?

**Comment by dynabler** [comment]
Any examples of scrapers that can do multi-mode?
